src/tasks/buildAndUpload.ts

```diff
@@ -168,25 +175,47 @@ as ${releaseFilesDefaultNames.avatar} and then remove the 'manifest.avatar' prop
       }
     },
 
+    {
+      title: "Validate files",
+      task: () => {
+        for (const [fileId] of Object.entries(releaseFiles)) {
+          switch (fileId as keyof typeof releaseFiles) {
+            case "setupWizard":
+              if (!setupWizardString) continue;
+              validateSchema(yaml.load(setupWizardString));
```

Comment:
What the reason for this? I don't follow

Reply:
fixed. Read in the block scope and then dropped